.github/workflows/course-schedule.yml

```diff
@@ -0,0 +1,25 @@
+name: "Course Schedule Updates"
+on:
+  - pull_request
+
+jobs:
+  course-schedule:
+    runs-on: ubuntu-latest
+    name: Make Course Schedule Comment
+    permissions:
+      pull-requests: write
```

Comment:
From [Permissions for the GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), I think this will be ineffective for PRs that come from forks.

I guess we'll figure it out soon after merging it :smile:

Reply:
Maybe I misunderstood earlier suggestions - does this not run when we click "Run Actions" for an untrusted fork?

Reply:
It does run, but it fails because the token is downgraded. That's how I interpret the error I just saw in #1693:

![image](https://github.com/google/comprehensive-rust/assets/89623/79b13816-4eed-4165-b3a3-60b05a6c5a54)

Reply:
This makes me think that we should remove this again, sorry!